custom/conf/app.ini.sample

```diff
@@ -141,8 +141,9 @@ KEYWORDS = go,git,self-hosted,gitea
 [markdown]
 ; Enable hard line break extension
 ENABLE_HARD_LINE_BREAK = false
-; List of custom URL-Schemes that are allowed as links when rendering Markdown
-; for example git,magnet
+; Comma separated list of custom URL-Schemes that are allowed as links when rendering Markdown
+; for example http,https,git,magnet,ftp (more at https://en.wikipedia.org/wiki/List_of_URI_schemes)
```

Comment:
```suggestion
; for example git,magnet,ftp (more at https://en.wikipedia.org/wiki/List_of_URI_schemes)
```
http and https should not be in the list to avoid confusion.